lib/openxml/docx/properties/level_text.rb

```diff
@@ -0,0 +1,14 @@
+module OpenXml
+  module Docx
+    module Properties
+      class LevelText < StringProperty
```

Comment:
According to the OpenXml spec (Pt. 1, 17.9.11), it looks like the `w:null` attribute won't have any effect if the `w:val` attribute is set; as a string property, `w:val` will _always_ be generated. As such, I wonder if it might make more sense here to have `LevelText` inherit from `ComplexProperty` and then include a custom initializer that assigns its argument to a `value` attribute if it's not null. You can still include it as a `value_property` in an element; that will ensure that the `level_text=` method is generated. So it might look something like this:

```ruby
...
class LevelText < ComplexProperty
  tag :lvlText

  with_namespace :w do
    attribute :null, expects: :boolean
    attribute :value, expects: :string, displays_as: :val
  end

  def initialize(value)
    self.value = value unless value.nil?
  end

end
...
```